tasks/tests.py

```diff
@@ -13,9 +15,28 @@
     ProjectCreateView,
     TaskUpdateView,
     TaskDeleteView,
+    TaskOverdueListView,
 )
 
 # Create your tests here.
+def create_project(
+    project_name="Deployment",
+):
+    project = Project.objects.create(name=project_name)
+    return project
+
+
+def create_task(task_text, project_name, completed=False):
+    today = now()
+    project = create_project(project_name)
```

Comment:
What will happen if I call `create_task` method  2 or more times? Will it create 2 or more projects?
It should not create multiple projects. project itself should be passed from parameter.